conf/env

```diff
@@ -0,0 +1,33 @@
+DB_SRC="3.108.123.110"
```

Comment:
Is this file used in any way to build the Pentaho container image? If not, please remove it from git.